### ISSUE TITLE
AP90, correction of loose IAST nasal spellings

### ISSUE THREAD

**Issue by funderburkjim** [sanskrit-lexicon/CORRECTIONS#328]
In the conversion to modern IAST discussed in #327 , an ambiguity was noticed in some aspects of the
italicized IAST.  For instance, under headword संज्ञा : 

![image](https://cloud.githubusercontent.com/assets/6393033/21532810/faeb6b7c-cd21-11e6-90ca-7e293eaae816.png)

Here the italicized `n` represents the palatal nasal ञ्   (or maybe it should be anusvAra, since part of
prefix सम् ) 

By contrast, italic `n` usually represents the cerebral nasal ण् .

In the change to modern IAST, ALL italic `n` were changed to n-with-underdot.  

So this word is now spelled (incorrectly) as Saṇjnā .

In other words, to get an accurate, modern IAST,  we would need to make a further examination
of cases.  For instance, the correct IAST spelling in this case is likely Saṃjñā 

This issue is opened to identify this as an area requiring further work.

When anyone wants to tackle this, the [findiast12_transcoded.txt](http://sanskrit-lexicon.uni-koeln.de/scans/AP90Scan/2014/pywork/20161228/findiast12_transcoded.txt) file  (field 4) will
be a good starting point.
